docker-compose.yml

```
@@ -85,7 +85,96 @@ services:
+      NEXT_PUBLIC_MEET_TYPE: ${NEXT_PUBLIC_MEET_TYPE:-jitsi}
+      NEXT_PUBLIC_MEET_DOMAIN: ${NEXT_PUBLIC_MEET_DOMAIN:-meet.ever.team}
+      LIVEKIT_API_SECRET: ${LIVEKIT_API_SECRET:-}
+      LIVEKIT_API_KEY: ${LIVEKIT_API_KEY:-}
+      NEXT_PUBLIC_LIVEKIT_URL: ${NEXT_PUBLIC_LIVEKIT_URL:-}
+      INVITE_CALLBACK_URL: ${INVITE_CALLBACK_URL:-https://app.ever.team/auth/accept-invite}
+      VERIFY_EMAIL_CALLBACK_URL: ${VERIFY_EMAIL_CALLBACK_URL:-https://app.ever.team/verify-email}
+      SMTP_PORT: ${SMTP_PORT:-587}
+      NEXT_PUBLIC_DISABLE_AUTO_REFRESH: ${NEXT_PUBLIC_DISABLE_AUTO_REFRESH:-false}
+      SENTRY_LOG_LEVEL: ${SENTRY_LOG_LEVEL:-error}
+      NEXT_PUBLIC_SENTRY_DSN: ${NEXT_PUBLIC_SENTRY_DSN:-}
+      NEXT_PUBLIC_SENTRY_DSN: ${NEXT_PUBLIC_SENTRY_DSN:-}
```

YAMLlint (1.37.1)

[error] duplication of key "NEXT_PUBLIC_SENTRY_DSN" in mapping

(key-duplicates)